### ISSUE TITLE
Optimize skill descriptions - Run skill-creator description loop on all 4 skills

### ISSUE THREAD

**Issue by ssilvius** [rafters-studio/rafters#1430]
## Goal

Run the description-optimization loop from `claude-plugins-official/skill-creator` against each of the four rafters-* skills, applying the `best_description` to each SKILL.md frontmatter. Target: each skill ≥85% should-trigger and <15% should-not-trigger on a 20-query eval set, run 3 times for variance.

## Exact Implementation Requirements

NO emoji anywhere

### Behavior Requirements

- Use the model that powers the live session (Claude Opus 4.7 1M context as of writing) — match what users actually experience
- Run 5 iterations max per skill via `scripts/run_loop.py`
- Eval set is split 60% train / 40% held-out test (script handles this); accept the description selected by *test* score, not train, to avoid overfitting
- Each skill's optimization runs in its own background task
- Final scores recorded in a single `docs/SKILLS_EVALS.md` document

### Workflow

1. For each skill, ensure `evals/triggers.json` exists with 20 queries (10 should-trigger, 10 should-not-trigger near-misses)
2. Run optimization loop:
   ```bash
   python -m scripts.run_loop \
     --eval-set plugin/skills/<skill-name>/evals/triggers.json \
     --skill-path plugin/skills/<skill-name>/ \
     --model claude-opus-4-7 \
     --max-iterations 5 \
     --verbose
   ```
3. Apply `best_description` from each loop's output JSON to the skill's SKILL.md frontmatter
4. Record before/after scores in `docs/SKILLS_EVALS.md` with iteration history

### Quantitative Thresholds

For each skill, on the held-out test set:
- should-trigger pass rate ≥85% (≥7 of 8 test queries trigger)
- should-not-trigger pass rate <15% false-positive (≤1 of 8 near-miss queries trigger)
- Each query run 3 times; pass rate is the average

### Required Files

- `docs/SKILLS_EVALS.md` — table of skill × iteration × (train_score, test_score), final selected description, before/after delta
- Each `plugin/skills/<skill-name>/evals/optimization-history.json` — full per-iteration record from the loop

### Validation Checklist

- [ ] All 4 skills meet thresholds
- [ ] Best descriptions applied to SKILL.md frontmatter (verified by `git diff`)
- [ ] Frontmatter still ≤1024 chars after optimization
- [ ] Loop ran with the session model, not a default
- [ ] Variance documented (mean ± stddev across 3 runs per query)

## What NOT to Include

- Trigger eval set authoring (those land with #4-7 — this issue assumes they exist)
- Pressure tests (covered in #4-7)
- Body text revisions to SKILL.md (only the description field is optimized here)
- Distribution wiring (#9)

## File Locations

- Eval results: `docs/SKILLS_EVALS.md`
- Per-skill history: `plugin/skills/<skill-name>/evals/optimization-history.json`
- Updated frontmatter in each existing `plugin/skills/<skill-name>/SKILL.md`

### Dependencies

- Requires `claude-plugins-official/skill-creator` to be installed (already cached at `~/.claude/plugins/cache/claude-plugins-official/skill-creator/`)
- Requires `claude` CLI for the loop's `claude -p` subprocess calls
- Requires Python 3.10+ (skill-creator scripts)

## Success Criteria

- [ ] All 4 skills ≥85% should-trigger, <15% should-not-trigger on held-out test
- [ ] `docs/SKILLS_EVALS.md` documents before/after for each
- [ ] Frontmatter still passes 1024-char limit
- [ ] CHANGELOG.md entry summarizing the optimization pass

**This issue is complete when:** all four `rafters-*` skills have optimized descriptions applied, evaluation thresholds are met on held-out test sets, and `docs/SKILLS_EVALS.md` documents the full before/after with iteration history.

## Context & References

- Spec: `docs/SKILLS_SPEC.md` (Testing section)
- Plan: `docs/SKILLS_PLAN.md` (Phase 5)
- Tooling: `~/.claude/plugins/cache/claude-plugins-official/skill-creator/scripts/run_loop.py`
- Depends on: #4, #5, #6, #7
- Blocks: #9
